LanguageApp/app/screens/ProviderScenarioScreen.js

```diff
@@ -40,22 +46,59 @@ function ProviderScenarioScreen({ route }) {
     setAnswerAudio(data);
   };
 
-  // Submission data to be passed to Firebase
-  const translatedScenario = {
-    promptTranslation: cpPrompt,
-    answerTranslation: cpAnswer,
-    promptRecording: promptAudioLink,
-    answerRecording: answerAudioLink,
-    translatorID: 11111111,
-  };
+    // Submission data to be passed to Firebase
+    const translatedScenario = {
+      promptTranslation: cpPrompt,
+      answerTranslation: cpAnswer,
+      promptRecording: promptAudioLink,
+      answerRecording: answerAudioLink,
+      translatorID: 11111111,
+    };
 
   const submitTranslation = async () => {
     console.log("submitting to database", translatedScenario);
-    console.log("Prompt audio uri to submit to Firebase", promptAudio);
-    console.log("Answer audio uri to submit to Firebase", answerAudio);
-    /* TO DO:  Add code to sumbit to Firestore */
-    setCpPrompt("");
-    setCpAnswer("");
+    console.log("Prompt audio uri to submit to storage??", promptAudio);
+    console.log("Answer audio uri to submit to storage...", answerAudio);
+
+    // Create calls to use to add to DB
+    // https://firebase.google.com/docs/firestore/manage-data/add-data#update_fields_in_nested_objects
+    var answerRecordingLanguage = "answerRecording." + route.params.language;
+    var answerTranslationLanguage = "answerTranslation." + route.params.language;
+    var promptRecordingLanguage = "promptRecording." + route.params.language;
+    var promptTranslationLanguage = "promptTranslation." + route.params.language;
+    var translatorIdLanguage = "translatorId." + route.params.language;
+    db.collection("Scenarios").doc(route.params.id).update({
+      [answerRecordingLanguage]: answerAudio, // needs to be changed to the storage link
+      [answerTranslationLanguage]: cpPrompt, 
+      [promptRecordingLanguage]: promptAudio, // needs to be changed to the storage link
+      [promptTranslationLanguage]: cpAnswer,
+      [translatorIdLanguage]: 12353464563
+    }).then(() => {
+        console.log(route.params.title, " scenario successfully updated!");})
```

Comment:
These logs remind me a bit of how you would interact with a CLI. In a CLI, the user is interacting directly with the command line. In this case, however, the user is interacting with the UI. It would be useful for the user to see some signifier of success like this in a message in the UI.